### PR TITLE
Installation review

### PR DESCRIPTION
**Pull request by hugovalente-pm**
##### Summary
Review the documentation to give more emphasis on using kickstart as part of the work on https://github.com/netdata/product/issues/1531

##### Component Name
Documentation

##### Test Plan

Review docs after merge

### CODE REVIEW

File: claim/README.md [hunk 1]
```diff
@@ -6,17 +6,17 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/claim/README.md
...
+You can securely connect a Netdata Agent, running on a distributed node, to Netdata Cloud. A Space's
 administrator creates a **claiming token**, which is used to add an Agent to their Space via the [Agent-Cloud link
 (ACLK)](/aclk/README.md).
 
 Are you just starting out with Netdata Cloud? See our [get started with
 Cloud](https://learn.netdata.cloud/docs/cloud/get-started) guide for a walkthrough of the process and simplified
 instructions.
 
-Connecting Agents, also referrered as **nodes**, is a security feature in Netdata Cloud. Through the connection process, you demonstrate in a few ways
+Connecting Agents, also referred as **nodes**, is a security feature in Netdata Cloud. Through the connection process, you demonstrate in a few ways
```
Comment (Ferroin) [inline claim/README.md:17]: ‘Connecting’ the agents is _NOT_ a security feature, the verification we do to establish proof of ownership is.

To anybody with basic knowledge of security this sounds like we have no idea what we’re talking about.

To someone who is paranoid about security and being tracked, this description is a _huge_ red flag.

This should probably instead be rephrased to something like:

‘When connecting an agent (also referred to as a node) to Netdata Cloud, you must complete a verification process that proves you have some level of authorization to manage the node itself. This verification is a security feature that helps prevent unauthorized users from seeing the data on your node.’

Comment (hugovalente-pm) [reply claim/README.md:17]: Accepted your suggestion @Ferroin , will put it on next update to the PR. thanks

File: claim/README.md [hunk 2]
```diff
@@ -41,44 +38,68 @@ There are two important notes regarding connecting nodes:
 
 ## How to connect a node
 
+There will be two main flows from where you might want to connect a node to Netdata Cloud.
```
Comment (manos-saratsis) [inline claim/README.md:41]: @hugovalente-pm Only admins can connect nodes for both scenarios. This needs to be mentioned

File: claim/README.md [hunk 3]
```diff
@@ -41,44 +38,68 @@ There are two important notes regarding connecting nodes:
...
+given by Netdata Cloud into your node's terminal.
 
-```bash
-sudo netdata-claim.sh -token=TOKEN -rooms=ROOM1,ROOM2 -url=https://app.netdata.cloud
+### Connect an agent running in Linux
+
+If you want to connect a node that is running on a Linux environment, the script that will be provided to you by Netdata Cloud is the [kickstart](/packaging/installer/#automatic-one-line-installation-script) which will install the Netdata Agent on your node, if it isn't already installed, and connect the node to Netdata Cloud. It should be similar to:
+
+```
+bash <(curl -Ss https://my-netdata.io/kickstart.sh) --claim-token TOKEN --claim-rooms ROOM1,ROOM2 --claim-url https://app.netdata.cloud
```
Comment (manos-saratsis) [inline claim/README.md:71]: @hugovalente-pm instead of TOKEN, ROOM1, ROOM2 I would add some sample values.

**Review by manos-saratsis — APPROVED** [review]

File: claim/README.md (hunk 3 above)
Comment (hugovalente-pm) [reply claim/README.md:71]: In this case I did not change what was the original one (examples https://learn.netdata.cloud/docs/agent/packaging/installer/methods/kickstart#connect-node-to-netdata-cloud-during-installation and https://learn.netdata.cloud/docs/agent/claim#how-to-connect-a-node) do you mean to change to some concrete example across all docs?

File: claim/README.md (hunk 2 above)
Comment (hugovalente-pm) [reply claim/README.md:41]: Will add that note, thanks.

File: claim/README.md (hunk 3 above)
Comment (hugovalente-pm) [reply claim/README.md:71]: @manos-saratsis won't change this for now, if we want we can revisit in a next iteration

**Comment by hugovalente-pm** [comment]
Will open a new PR

**Pull request closed without merging**